src/components/interfaces/MOBILE_API.xml

```diff
@@ -3060,16 +3060,20 @@
         <param name="hdRadioEnable" type="Boolean" mandatory="false"  since="5.0">
             <description> True if the hd radio is on, false if the radio is off</description>
         </param>
-        <param name="availableHDs" type="Integer" minvalue="1" maxvalue="7" mandatory="false" since="5.0">
-            <description>number of HD sub-channels if available</description>
+        <param name="availableHDs" type="Integer" minvalue="0" maxvalue="7" mandatory="false" deprecated="true" since="5.1">
+            <description>Number of HD sub-channels if available</description>
             <history>
                 <param name="availableHDs" type="Integer" minvalue="1" maxvalue="3" mandatory="false" since="4.5" until="5.0"/>
             </history>
         </param>
-        <param name="hdChannel" type="Integer" minvalue="1" maxvalue="7" mandatory="false" since="5.0">
+        <param name="availableHdChannels" type="Integer" minvalue="0" maxvalue="7" array="true" minsize="0" maxsize="8" mandatory="false" since="5.1">
+            <description>The list of available HD sub-channel indexes. Empty list means no Hd channel is available. Read-only. </description>
+        </param>
+        <param name="hdChannel" type="Integer" minvalue="0" maxvalue="7" mandatory="false" since="5.1">
             <description>Current HD sub-channel if available</description>
             <history>
                 <param name="hdChannel" type="Integer" minvalue="1" maxvalue="3" mandatory="false" since="4.5" until="5.0"/>
+                <param name="hdChannel" type="Integer" minvalue="1" maxvalue="7" mandatory="false" since="5.0" until="5.1"/>
```

Comment:
@IGapchuk why until "5.1"? I think here should be just `since="5.1."`

Reply:
@AKalinich-Luxoft parameter 'hdChannel' had minvalue="1" and maxvalue="3" from 4.5 into 5.0 versions, after that it had minvalue="1" and  maxvalue="7" since 5.0 version. And since 5.1 API version this parameter will have minvalue="0" and maxvalue="7".